src/java.desktop/share/classes/java/awt/Robot.java

```diff
@@ -189,6 +226,11 @@ private static void checkIsScreenDevice(GraphicsDevice device) {
 
     /**
      * Moves mouse pointer to given screen coordinates.
+     * @implNote the mouse pointer may not visually move on some platforms,
```

Comment:
This doesn't seem to be an implNote in the way that the JEP
intends since it has nothing to do with anything we did in our implementation. I think it would have to be just spec, in which case I am not sure we should call out Linux+Wayland

See  https://openjdk.org/jeps/8068562

"Implementation Notes. This section contains informative notes about the implementation, such as advice to implementors, or performance characteristics that are specific to the implementation in this class of this version of the JDK. The information in this section is subject to change from release to release. These characteristics are also allowed to vary across platforms, vendors, and versions."

Reply:
Updated.